config/prompts_sys.py

```
@@ -1431,6 +1431,64 @@ def _normalize_prompt_language(lang: str) -> str:
+    'en': """Please talk to {master_name} naturally in your character. Requirements:
+1. Fully match your character's personality and speaking habits
+2. Be brief and natural, like casual sharing or starting a conversation, no more than 2-3 sentences
+3. Fit the current conversation atmosphere and master's recent interests
+4. Absolutely do NOT repeat content from "Recent Chat Records". Repetition is judged strictly: if the core event/person/video/meme is the same, even with different wording, tone, or angle, it's considered repetition and you must give up
+5. Do not repeat your own recent proactive chats: do not mention the same news, video, dispute point, or joke again; if you can't confirm whether it's repetitive, treat it as repetitive and give up
+6. If there's any risk of repetition, prefer replying [PASS] rather than forcing a chat
+7. If the provided materials are all unsuitable for chatting (too boring, repeated recently, or can't find a natural angle), directly reply [PASS]
+8. Do not generate thinking processes
+{music_instruction}
+{meme_instruction}""",
+    'ja': """あなたのキャラクターとして、{master_name}に自然に話しかけてください。要求：
```

Ruff (0.15.6)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)